tests/testthat/test-trailing_blank_lines_linter.R

```diff
@@ -96,3 +98,55 @@ test_that("returns the correct linting", {
     column_number = 1L
   ), linter)
 })
+
+test_that("blank lines in chunks produce lints", {
+  linter <- trailing_blank_lines_linter()
+
+  tmp6 <- withr::local_tempfile(fileext = ".Rmd")
+  cat(
+    '---
+      title: "Some file"
+      ---
+
+      ```{r}
+      abc = 123
+
+      ```
+      \n',
+    file = tmp6
+  )
+
+  expect_lint(
+    content = NULL,
```

Comment:
why content= NULL?

Reply:
> file: if not `NULL`, read content from the specified file rather than from `content`.

Because we are specifying `file` argument. This is true across all tests in this file.

Reply:
hmm I don't read that the same. it's if file is not NULL, doesn't say anything about the nullity of content?

Reply:
`content` is no longer `NULL`.